server/index.js

```diff
@@ -23,13 +26,36 @@ async function start() {
 
   try {
     app.use(cors(corsOptions))
+    app.use(cookieParser())
+    app.use(express.json())
```

Comment:
@AStashevskaya It this middleware duplicated `bodyParser.json({ extended: true })` ?